src/sentry/search/utils.py

```diff
@@ -274,7 +282,7 @@ def parse_query(project, query, user):
                 results['bookmarked_by'] = parse_user_value(value, user)
             elif key == 'subscribed':
                 results['subscribed_by'] = parse_user_value(value, user)
-            elif key == 'first-release':
+            elif key in ('first-release', 'firstRelease'):
```

Comment:
@TheSavior mostly thinking around standardization. We have some _ and some -, and most things we use camelCase when possible (or dot to separation subsets).